source/components/GlobalNavigation/__snapshots__/index.spec.js.snap

```diff
@@ -546,7 +546,7 @@ exports[`render GlobalNavigation renders correctly when user or partnerSlot are
           isOpen={false}
           openModal={[Function]}
         >
-          <Wrapper
+          <withI18nextTranslation(Search)
```

Comment:
How did this happen?

Reply:
I'd guess it's because i18next lib was installed (because it's been moved from peerDeps to deps).